mobile/library/common/http/client.h

```diff
@@ -202,12 +204,15 @@ class Client : public Logger::Loggable<Logger::Id::http> {
     absl::optional<envoy_error> error_;
     bool success_{};
 
-    // Buffered response data when in explicit flow control mode.
+    // Buffered response data when in explicit flow control or buffering chunks.
     Buffer::InstancePtr response_data_;
     ResponseTrailerMapPtr response_trailers_;
     // True if the bridge should operate in explicit flow control mode, and only send
     // data when it is requested by the caller.
     bool explicit_flow_control_{};
+    // If greater than zero, indicates the minimum size of data that should be
+    // sent without end stream.
```

Comment:
nit: "sent" is a bit ambiguous here. I think we're talking about response data that we're receiving from the server and sending up to the bridge. But "sent" might also apply to request data we're sending to the server.